portfolio/src/main/java/com/google/sps/servlets/BlogCommentServlet.java

```diff
@@ -152,5 +162,53 @@ private String getUploadedFileUrl(HttpServletRequest request, String formInputEl
       return imagesService.getServingUrl(options);
     }
   }
+
+  /* returns the sentiment score of the message */
+  private float getSentimentScore(String message) {
```

Comment:
You have the same method in Blog and comment services. This is an opportunity to refactor them into something like a separate `LanguageService`.

Reply:
@jsheng112 I don't think this method is called anywhere. You probably just forget to delete it from here after moving it to `CommentService`.